src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,56 @@
+.stopwatch {
+  $stopwatch-radius: 40vh;
+  $stopwatch-border-width: 0.25vh;
+  $stopwatch-center-radius: $stopwatch-radius * 0.05;
+  $stopwatch-minute-hand-length: $stopwatch-radius * 0.7;
+  $stopwatch-minute-hand-width: $stopwatch-center-radius * 0.5;
+  $stopwatch-second-hand-length: $stopwatch-radius * 0.9;
+  $stopwatch-second-hand-width: $stopwatch-center-radius * 0.1;
+
+  &__container {
+    position: relative;
+
+    @extend %flexCenterXY;
+    @include circle($stopwatch-radius);
+
+    border: $stopwatch-border-width solid $accent-black-color;
+    background-color: $accent-light-gray-color;
+  }
+
+  &__center {
+    @include circle($stopwatch-center-radius);
+    background-color: $accent-black-color;
+  }
+
+  &__minute-hand {
+    @extend %handsPositioning;
+
+    width: $stopwatch-minute-hand-width;
+    height: $stopwatch-minute-hand-length;
+
+    background-color: $accent-black-color;
+
+    @extend %handsTransformSettings;
+
+    animation: handRotation 3600s linear 0s infinite;
```

Comment:
I think it`s not necessary to set animation-delay value in this property, as 0s is initial.